__tests__/app.test.js

```diff
@@ -146,4 +146,69 @@ describe("GET /api/articles/:article_id/comments",()=>{
             expect(res.body.msg).toBe("Not Found");
         });
     })
+})
+describe("POST /api/articles/:article_id/comments",()=>{
+    test("responds with status 201, posts the new comment and sends the new comment back to the client",()=>{
+        const newComment = {
+            username: "lurker",
+            body: "I cant code when i am sleepy."
+        }
+        return request(app)
+        .post('/api/articles/2/comments')
+        .send(newComment)
+        .expect(201)
+        .then(({body})=>{
+            expect(body.comment).toMatchObject({
```

Comment:
We would want this to be a little more specific as we know what the article id is and author etc

Reply:
i was told yesterday toMatchObject is more flexible even though i was just getting a single object back (and i was checking for all values in object). So i started using toMatchObject in any test case for objects😬

should i keep switching as per feedback?😂